src/instructlab/lab.py

```diff
@@ -15,6 +15,7 @@
 # NOTE: Subcommands are using local imports to speed up startup time.
 from .config import config as config_group
 from .data import data as data_group
+from .debug import debug
```

Comment:
That's fair, I changed it to `system` per @n1hility's recommendation, since `ilab system reset` implies resetting the config. `ilab storage` makes a lot of sense though, since we'll also probably need a way to view the total storage usage with a command like `ilab storage view` and `ilab storage clear`. 

We can make a dev-doc for this down the road though.